### ISSUE TITLE
Version 2.0.0 is only compatible with IntelliJ 2017.3

### ISSUE THREAD

**Issue by alexbde** [jimschubert/intellij-openapi-generator#20]
Hello,

I just wanted to install your plugin 2.0.0 in IntelliJ IDEA 2018.2.1. Unfortunately this is not possible because compatibility states "173.4301—173.*". Is this on purpose?

Thank very much in advance!

**Comment by jimschubert** [comment]
I don't recall changing the supported versions. I'll look into what happened and see if I can get a new version back to wider IntelliJ support. Thanks for reporting it!

**Comment by jimschubert** [comment]
I pushed version 2.0.1, bumping OpenAPI Generator to version 3.2.0 just to see if there was an issue with how I published v2.0.0; I used "Update Plugin" via web rather than command line as I normally do.

This didn't resolve the version issue, so I did a little more investigation and found that I had set `updateSinceUntilBuild true` in build.gradle. This makes plugin.xml match the idea version defined in build.gradle, which was 2017.3.3. I don't recall doing this, I may have accidentally done it when I moved the plugin to a subproject. I pushed 2.1.0 as well, which should fix this issue. Sorry about that!

Both versions 2.0.1 and 2.1.0 are pending. For 2.0.0, it took about 72 hours for Jetbrains to approve.

![image](https://user-images.githubusercontent.com/109659/44006658-b7eab02e-9e55-11e8-98a6-563a53f08d68.png)

I'll close this when I get notified of plugin approval.
